talent/sourcerers/README.md

```diff
@@ -9,5 +9,6 @@ We have created a list of the questions and below are the instructions on how to
 - Copy the content of the [questions](questions.md)
 - Create a new file (your_GitHub_username.md) in a new branch with your responses on the template you copied and create a Pull Request
 - Your file should be placed in this folder
+- You can merge your commit(s) after your answers have been approved by one member 
```

Comment:
Since opened PRs have more visibility than closed ones, we decided to keep minutes PRs opened for a while, to let other people to read them.
Since the goal of these Q&A is to let the people to know about their mates, would it be a good idea to apply that suggestion here too?